### PR TITLE
docs: add lambda plugin policy

### PR DESCRIPTION
**Pull request by ravikp7**
- add lambda page under policy section
- add lambda icon under Utilities in resources page
- replace resource icons with improved ones

### CODE REVIEW

File: _pages/resources.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ layout: page
 
 #### Utilities
 <ul class="icon-list">
+<li>{% include res-lambda.svg %}<h3>Lambda</h3><p>Proxy to AWS Lambda Functions</p></li>
```
Comment (altsang) [inline _pages/resources.md]: let's put the lambda plugin into the `Customization` category

**Review by altsang — DISMISSED** [review]
let's change the category, but otherwise LGTM

File: _pages/resources.md (hunk 1 above)
Comment (ravikp7) [reply _pages/resources.md]: @altsang Done!

**Review by XVincentX — APPROVED** [review]

**Comment by XVincentX** [comment]
For some reason the CI does not trigger when you open the PRs.

**Pull request closed without merging**